### DISCUSSION TITLE
Starting the pipeline with a count matrix

### DISCUSSION THREAD

**Discussion by lauracubitt** [rajewsky-lab/spacemake#116] [Q&A] [answered]
Hi! I was wondering whether it's possible to start the snakemake pipeline at the point of an already processed count matrix? I have some pre-processed spatial data already, and was wondering if it were possible to start with this instead of the fastqs? Thank you!

**Comment by nukappa** [comment, marked as answer]
Hi @lauracubitt , it is also possible to provide a count matrix during setting up of the samples. I believe this has to be a raw count matrix and not normalized. See also [here](https://spacemake.readthedocs.io/en/latest/projects/index.html). Note that we haven't tested this feature for quite some time, so if you encounter unexpected behavior please let us know.

**Reply by lauracubitt** [reply]
Ah I hadnt spotted that option - thank you! I'll let you know if I come across any unexpected problems. Thanks for the quick reply!
